lib/nadia.ex

```diff
@@ -330,6 +330,23 @@ defmodule Nadia do
   def get_file(file_id), do: request("getFile", file_id: file_id)
 
   @doc """
+  Use this method to get link for file for subsequent use.
+  This method is an extension of the `get_file` method.
+
+  *Attention: link will be valid for 1 hour.*
+
+  Args:
+  * `file_id` - File identifier to get link
+  """
+  @spec get_file_link(binary) :: binary | {:error, Error.t}
+  def get_file_link(file_id) do
+    with {:ok, f} <- request("getFile", file_id: file_id),
```

Comment:
since `nadia` has a `get_file/1` already, I don't think this is a good way to implement `get_file_link/1`.

I prefer to something like this:
```elixir
@spec get_file_link(File.t) :: {:ok, binary} | {:error, Error.t}
```

`File.t` has not only the `file_path`, but also `file_size`.
For users of `nadia`, `file_size` may be a necessary part for deciding whether download it or not, and they can handle it easily using pattern matching.
so the use-case may be like this: 
get_file -> match the File -> get_file_link